modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1159,6 +1164,38 @@ private void _validateClientExtension(
 				clientExtension, "membershipType", "open", "private",
 				"restricted");
 		}
+
+		if (Objects.equals(clientExtension.type, "themeCSS")) {
+			String frontendTokenDefinitionFilePath =
+				clientExtension.typeSettings.getOrDefault(
+					"frontendTokenDefinition", ""
+				).toString();
+
+			if (!frontendTokenDefinitionFilePath.isEmpty()) {
+				_validateFrontendTokenDefinitionFile(
+					frontendTokenDefinitionFilePath, project);
+			}
+		}
+	}
+
+	private void _validateFrontendTokenDefinitionFile(
+		String filePath, Project project) {
+
+		try {
+			JsonParser.parseReader(new FileReader(project.file(filePath)));
+		}
+		catch (FileNotFoundException fileNotFoundException) {
+			throw new GradleException(
+				String.format(
+					"File %s not found or it is not a file.",
+					StringUtil.quote(filePath)));
+		}
+		catch (JsonParseException jsonParseException) {
+			throw new GradleException(
+				String.format(
+					"File %s does not contain a valid json content.",
```

Comment:
I guess "%s is not a valid JSON file" is a better message. But I don't have any reference for that, just an opinion :D